src/sparseml/modifiers/quantization/gptq/base.py

```diff
@@ -50,6 +50,7 @@ class GPTQModifier(Modifier):
             - LayerCompressor.revert_layer_wrappers()
 
 
+    :param actorder: Whether to use activation reordering or not
```

Comment:
Could you provide a short description of what activation reordering is? Also a nitpick but since this is a user facing parameter I think calling it activation_reordering would be better

Reply:
Ok tests yeah i agree - for param name it comes from gptq code 
https://github.com/AutoGPTQ/AutoGPTQ/blob/ea829c7bbe83561c2b1de26795b6592992373ef7/auto_gptq/quantization/gptq.py#L67
Other parts of the code is very similar to this code. I'm more inclined to keep the name as is for now